Flutter/DateRangePicker/customizations.md

```diff
@@ -66,6 +66,15 @@ Widget build(BuildContext context) {
 
 ![Customizations Date Range Picker](images/customizations/customizations.png)
 
+>**NOTE**
+* Priority order
```

Comment:
Cell decoration will be rendered in this order blackout dates, special dates, disable dates, today date, weekend dates,  trailing and leading dates and dates decoration based on the decoration given

Reply:
I have checked and changed it like this. I have updated in my latest commit.